integration/app/todo.state.ts

```diff
@@ -33,8 +34,21 @@ export class TodoState {
   }
 }
 
+export class SetPrefix {}
+
 @State<TodoStateModel>({
   name: 'todos',
+  defaults: {
+    todo: [],
+    pizza: { model: undefined }
+  },
   children: [TodoState]
 })
-export class TodosState {}
+export class TodosState {
+  @Action(SetPrefix)
+  setPrefix({ getState, setState, patchState }) {
+    const state = getState();
```

Comment:
Remove unused function

Reply:
do u mean setPrefix? The action setPrefix is used in app.component.ts?